pkg/data_sources/job.go

```diff
@@ -36,6 +36,12 @@ var jobSchema = map[string]*schema.Schema{
 		Computed:    true,
 		Description: "ID of the job this job defers to",
 	},
+	"self_deferring": &schema.Schema{
+		Type:        schema.TypeBool,
+		Computed:    true,
+		Default:     false,
```

Comment:
tests are complaining that this can't default as it's computed anyway in the data source 😅